bigbluebutton-html5/private/config/settings.yml

```diff
@@ -700,7 +700,7 @@ public:
   presentation:
     allowDownloadable: true
     panZoomThrottle: 32
-    restoreOnUpdate: false
+    restoreOnUpdate: true
```

Comment:
In general we avoid tweaking settings.yml during fixes but to be fair, I was have recently heard a few people requesting this as a new feature, not knowing it's an existing parameter. I'd be supporting committing this particular change.